File: hw/cañarte/u1/HW09CDToJavaScripAndPhyton/ec_edu_espe_farmsimulator_view/FarmSimulator.py
import datetime
import sys
import os

sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from ec_edu_espe_farmsimulator_model.Chicken import Chicken

class FarmSimulator:
    def main():
        print("Saray's Chicken Farm Simulator V 0.5")

        chicken_1 = Chicken(1, "Lucy", "white and brown", 0, True, datetime.date.today())
        print(f"Chicken ---> \n", chicken_1)

        id = 2
        name = "Maruja"
        color = "black"
        age = 1
        molting = False
        born_on_date = datetime.date.today()

        chicken_2 = Chicken(id, name, color, age, molting, born_on_date)
        print("Chicken ---> \n", chicken_2)

if __name__ == "__main__":
    FarmSimulator.main()

File: hw/bonifaz/u1/hw08ChickenFarmSimulatorjs/ChickenFarmSimulator/FarmSimulator.js
const Chicken = require('./Chicken');

class FarmSimulator {
    static main() {
        console.log("Christian's Chicken farm simulator V0.9");

        let chicken = new Chicken(1, "Luccy", "White and brown", 0, true, new Date());
        console.log("Chicken -->\n" + chicken.toString());

        let id = 2;
        let name = "Marujaa";
        let color = "black";
        let age = 1;
        let molting = false;
        let bornOnDate = new Date();

        chicken = new Chicken(id, name, color, age, molting, bornOnDate);
        console.log("Chicken 2 --> \n" + chicken.toString());
    }
}

FarmSimulator.main();
